documentation/book/con-cluster-operator-rbac.adoc

```diff
@@ -64,46 +64,66 @@ The Cluster Operator will not run under the cluster admin account.
 
 The `ClusterRoles` follow the _principle of least privilege_ and contain only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect, and Zookeeper clusters. The first set of assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as `StatefulSets`, `Deployments`, `Pods`, and `ConfigMaps`.
 
+Cluster Operator is using two different `ClusterRoles`.
```

Comment:
change it to: Cluster Operator uses `ClusterRoles` to grant permission at the namespace-scoped resources level and cluster-scoped resources level: 

    ClusterRole with namespace-scoped resources for the Cluster Operator

    link:examples/install/cluster-operator/020-ClusterRole-strimzi-cluster-operator-role.yaml[]

    clusterRole with cluster-scoped resources for the Cluster Operator

    link:examples/install/cluster-operator/021-ClusterRole-strimzi-cluster-operator-role.yaml[]